templates/systemd.service.j2

```diff
@@ -1,4 +1,8 @@
+# Managed by ansible (role: haproxy)
+# DO NOT EDIT!
+
 [Service]
+LimitNOFILE={{ haproxy_maxconn + 10 }}
```

Comment:
10 seems a little low - stdin, out, error, logfile and control socket are already 5 FDs, can we also make this configurable and default to e.g. 20 or something?